platformio/commands/account/command.py

```diff
@@ -194,5 +190,85 @@ def account_update(current_password):
 @cli.command("show", short_help="PIO Account information")
 @click.option("--offline", is_flag=True)
 @click.option("--json-output", is_flag=True)
-def account_show(**kwargs):
-    pioplus_call(sys.argv[1:])
+def account_show(offline, json_output):
+    client = AccountClient()
+    try:
+        info = client.get_account_info(offline)
+        if json_output:
+            return click.echo(json.dumps(info))
+        click.echo()
+        if info.get("profile"):
+            print_profile(info["profile"])
+        if info.get("packages"):
+            print_packages(info["packages"])
+        if info.get("subscriptions"):
+            print_subscriptions(info["subscriptions"])
+        return click.echo()
+    except exception.AccountNotAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+def print_profile(profile):
+    click.echo("Profile:")
+    if profile.get("username"):
+        click.echo("- Username: %s" % profile["username"])
+    if profile.get("email"):
+        click.echo("- Email: %s" % profile["email"])
+    if profile.get("first_name"):
+        click.echo("- First name: %s" % profile["first_name"])
+    if profile.get("last_name"):
+        click.echo("- Last name: %s" % profile["last_name"])
```

Comment:
Please keep original CLI style https://github.com/platformio/platformio-core/blob/develop/platformio/commands/device/command.py#L72
![Screen Shot 2020-04-17 at 13 18 49](https://user-images.githubusercontent.com/1760209/79559196-fbc65c80-80ad-11ea-8d9b-0c5d57020841.png)

We also use `tabulate` module https://github.com/platformio/platformio-core/blob/develop/platformio/commands/boards.py#L18